Makefile

```
@@ -1,9 +1,10 @@
 RUNNER_VERSION ?= 2.332.0
+JIT_RUNNERS_VERSION ?= $(shell git describe --tags --always 2>/dev/null || echo "dev")
 
 AMI_DISTRIBUTION_REGIONS ?= us-east-1 us-west-1 us-west-2 eu-west-1 eu-west-2 eu-west-3 eu-central-1 eu-north-1 sa-east-1
 SOURCE_REGION ?= us-east-2
 
-.PHONY: help test lint build clean lambda.build lambda.test ami.build ami.validate ami.distribute ami.copy
+.PHONY: help test lint build clean lambda.build lambda.test ami.build ami.build-test ami.validate ami.distribute ami.copy
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)